components/list/README.md

```diff
@@ -158,6 +158,12 @@ The `d2l-list` is the container to create a styled list of items using `d2l-list
 
 The `d2l-list` supports selectable items within a list, including both single and multi selection. Selection is enabled when `d2l-list-item`s have the `selectable` attribute. When items are selectable, multiple selection is the default behaviour, however the `selection-single` attribute can be applied to the `d2l-list` to enable single selection. A `d2l-list-header` component can be added to `d2l-list`'s `header` slot to provide select-all and bulk actions.
 
+## Pageable Lists
+
+Load-More paging functionality can be implemented in lists by placing a `d2l-pager-load-more` in `d2l-list`'s `pager` slot. The consumer must handle the `d2l-pager-load-more` event by loading more items, updating the pager state, and signalling completion by calling `complete()` on the event detail. Focus will be automatically moved on the first new item once complete. See [Paging](../../components/paging) for more details.
```

Comment:
Curious what people's thoughts are - I would like to have a full working Daylight demo for the load-more paging, bit there's a fair amount of extra coding that's needed to set it up, and eventually `d2l-table-wrapper` will also support load-more, so perhaps it makes sense to place the demo in the Paging readme, but then it's not in the list/table documentation where people might look for it.